### DISCUSSION TITLE
Display name replacement

### DISCUSSION THREAD

**Discussion by GoofyGuyWhoCantAnimate** [anthony1x6000/ROBLOX2016stylus#48] [Ideas]
Ive never liked the new "display names" feature thing and always wanted to disguise them. All it would have to do is simply: remove the "@ section" and REPLACE the text in the display name section with what the @ would have said.

Before:
![image](https://github.com/anthony1x6000/ROBLOX2016stylus/assets/91696629/f239fe51-eb4a-435b-a15c-7ad46ad1c060)

After:
![image](https://github.com/anthony1x6000/ROBLOX2016stylus/assets/91696629/3f330e7e-91a5-4d54-932f-4566f76e2ade)

I know the theme was originally just to make it look the same, rather than make it seem like it really is 2016, but yeah. As said it could always be an extra thing like the "Hello, username!" gui.

Thanks.

EDIT: (this is an unrelated issue, It didn't feel right to have 2 issues open at the same time and I'm worried I won't remember later, so here it is) 
I've also noticed, a few moments after posting, that the profile pictures are also wrong for 2016. 
They should be identical to (by this, I mean have the same imageURL as) the full-body avatar previews below the main profile picture. Maybe that could also be added in the future?

**Comment by anthony1x6000** [comment]
I may implement an optional removal of display names later. However, it would be a relatively low priority, as I'd like to retain the full functionality of the website. 
I'm also curious about the issue you mentioned about the profile pictures.
Do you mean by having the character looking slightly to the right, similar to the following image?
![image](https://github.com/anthony1x6000/ROBLOX2016stylus/assets/33004321/216c42c3-d99b-4697-92c6-531d9bd1ed38)
Or like cropping a full-size avatar into the profile picture space?
![image](https://github.com/anthony1x6000/ROBLOX2016stylus/assets/33004321/9504eabb-74e8-4612-ab52-b9b43a6079b4)

**Reply by GoofyGuyWhoCantAnimate** [reply]
The second one.

![image](https://github.com/anthony1x6000/ROBLOX2016stylus/assets/91696629/7dc495e2-e61d-4b88-9c2f-285d68d6a4fc)

I was slightly wrong, it's not like this for your main profile, but it IS like that for every OTHER appearance (e.g. friends list on someone else's profile, homepage, user search, etc)

**Reply by anthony1x6000** [reply]
Oh, I see. Yeah, that doesn't seem too hard to implement through javascript. Once I get some free time, I'll try working on that. 
Thanks for the suggestion, never thought of implementing that before.
